### ISSUE TITLE
Construct heading sequentially

### ISSUE THREAD

**Issue by caseyedavis12** [WGBH/AAPB2#1085]
Headings should be constructed sequentially. There should only be one h1 per page. 

On homepage, AAPB logo should be the h1
On other pages, the h1 should be the text in the  purple banner (Search, Records, About the AAPB, etc.)
The h2 should be the Titles on search results and record pages.
The h3 should be the Limit Your Search header.

**Comment by sroosa** [comment]
@jasoncorum did this get addressed in the pr mentioned above? Should the ticket get closed too?
